cloudsync_onedrive.py

```diff
@@ -39,11 +40,27 @@
 
 import quickxorhash
 
-__version__ = "2.1.1" # pragma: no cover
+__version__ = "2.2.0" # pragma: no cover
 
 
 SOCK_TIMEOUT = 180
 
+
+class EventFilter(enum.Enum):
```

Comment:
Do we expect this to change provider to provider?

Reply:
I have it in here for now because I'm not sure if other providers will need it -- most (if not all) other providers support server-side event filtering, and if that works as expected then we don't need to do any special filtering for them. If not, then it would make sense to move this to the cloudsync library and have all providers pull it in from there.